tsconfig.json

```diff
@@ -0,0 +1,12 @@
+{
+  "compilerOptions": {
+    "module": "commonjs",
+    "target": "es5",
```

Comment:
not es6?

Reply:
Left it as es5 for backwards compatibility (since I saw you downgraded your source to es5 in a commit). Happy to change back to es6 if not a big deal. This actually may help reduce the deliverable size if it uses less polyfill